hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -807,6 +807,30 @@ def _track_subtype(self, init_values):
                 return self.env.ref("hr_timesheet_sheet.mt_timesheet_approved")
         return super()._track_subtype(init_values)
 
+    # ------------------------------------------------
+    # to prevent sending emails to the reviewer
+    # ------------------------------------------------
+
+    def _notify_compute_recipients(self, message, msg_vals):
```

Comment:
Docstring and description about function add please.

Reply:
resolved

Reply:
> Docstring and description about function add please.

Are you shore that we need to put a new docstring for an inherited function?

Reply:
what solution is right?